ERROR - 2025-04-30 09:09:15 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(62): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:09:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:09:15 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:15\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:15\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:09:15 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:15\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:15\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:15\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:09:44 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(62): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:09:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:09:44 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:43\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:43\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:09:44 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:43\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:43\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:43\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:09:53 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(62): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:09:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:09:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:53\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:53\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:09:53 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:53\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:53\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:09:53\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:10:25 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(62): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:10:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:10:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:25\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:25\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:10:25 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:25\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:25\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:25\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:10:47 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(62): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:10:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:10:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:47\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:47\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:10:47 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:47\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:47\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:10:47\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:11:47 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(62): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:11:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:11:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:11:47\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:11:47\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:11:47 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:11:47\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:11:47\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:11:47\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:14:03 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(62): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:14:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:14:03 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:14:03\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:14:03\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:14:03 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:14:03\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:14:03\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:14:03\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(62): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:29:41 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(63): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:29:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:29:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'Bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'e10adc3949ba59abbe56e057f20f883e\', \'2025-04-30 09:29:40\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'Bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'e10adc3949ba59abbe56e057f20f883e\', \'2025-04-30 09:29:40\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:29:41 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'Bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'e10adc3949ba59abbe56e057f20f883e\', \'2025-04-30 09:29:40\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'Bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'e10adc3949ba59abbe56e057f20f883e\', \'2025-04-30 09:29:40\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'0\', \'Bala\', \'bala@gmail.com\', \'bala1@gmail.com\', 1, 2, \'e10adc3949ba59abbe56e057f20f883e\', \'2025-04-30 09:29:40\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Id`, `us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Id:, :us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:31:50 --> ParseError: syntax error, unexpected token "=>"
[Method: POST, Route: staff/save]
in APPPATH\Controllers\Staff.php on line 59.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\ZakhiDesigns\\app\\Controllers\\Staff.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Staff')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Staff')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Staff', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:31:56 --> ParseError: syntax error, unexpected token "=>"
[Method: POST, Route: staff/save]
in APPPATH\Controllers\Staff.php on line 59.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\ZakhiDesigns\\app\\Controllers\\Staff.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Staff')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Staff')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Staff', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:31:59 --> ParseError: syntax error, unexpected token "=>"
[Method: GET, Route: staff/add]
in APPPATH\Controllers\Staff.php on line 59.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\ZakhiDesigns\\app\\Controllers\\Staff.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Staff')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Staff')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Staff', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:32:53 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(63): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:32:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:32:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:53\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:53\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:32:53 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:53\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:53\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:53\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:32:56 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(63): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:32:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:32:56 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:56\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:56\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:32:56 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:56\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:56\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:32:56\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-30 09:33:05 --> mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list' in C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\ZakhiDesigns\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\ZakhiDesigns\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\ZakhiDesigns\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\ZakhiDesigns\app\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\wamp64\www\ZakhiDesigns\app\Controllers\Staff.php(63): App\Models\StaffModel->createStaff(Array)
#6 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(933): App\Controllers\Staff->createnew()
#7 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
#8 C:\wamp64\www\ZakhiDesigns\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\ZakhiDesigns\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\ZakhiDesigns\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\wamp64\www\ZakhiDesigns\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-30 09:33:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
[Method: POST, Route: staff/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 2 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:33:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:33:05\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:33:05\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 4 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-30 09:33:05 --> [Caused by] mysqli_sql_exception: Unknown column 'us_Created_on' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:33:05\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:33:05\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (\'bala\', \'bala@gmail.com\', \'\', 1, 2, \'ab9731f038233e43a3d2f47ec5d8cee9\', \'2025-04-30 09:33:05\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`us_Name`, `us_Email`, `us_Email2`, `us_Status`, `us_Role`, `us_Password`, `us_Created_on`, `us_Created_by`, `us_Modified_by`) VALUES (:us_Name:, :us_Email:, :us_Email2:, :us_Status:, :us_Role:, :us_Password:, :us_Created_on:, :us_Created_by:, :us_Modified_by:)', [...], false)
 5 APPPATH\Models\StaffModel.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Staff.php(63): App\Models\StaffModel->createStaff([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Staff->createnew()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Staff))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
